src/simulation/dynamics/DynOutput/boreAngCalc/boreAngCalc.cpp

```diff
@@ -140,6 +140,17 @@ void BoreAngCalc::computeCelestialOutputData()
 */
 void BoreAngCalc::computeInertialOutputData()
 {
+    // Compute the DCM from inertial do body frame
+    Eigen::MRPd sigma_BN = cArray2EigenMRPd(this->localState.sigma_BN);
+    Eigen::Matrix3d dcm_BN = sigma_BN.toRotationMatrix().transpose();  /*!< dcm, inertial to body frame */
+
+    // Calculate the inertial heading vector
+    Eigen::Vector3d inertialHeadingVec_B = dcm_BN * this->inertialHeadingVec_N;
+    double dotValue = this->boreVec_B.dot(inertialHeadingVec_B);
+    this->boresightAng.missAngle = fabs(safeAcos(dotValue));
+
+    // Azimuth is undefined, so we set it to 0
+    this->boresightAng.azimuth = 0.0;
```

Comment:
:small_red_triangle_down: I see the intention and it's good thinking. But in this case when you add a function, add the body in the same commit. You can squash this commit into the previous.

Reply:
Done